test/functional/bulk_tests.js

```diff
@@ -73,6 +73,29 @@ describe('Bulk', function() {
     }
   });
 
+  it('should use arrayFilters for updateMany', function(done) {
+    const configuration = this.configuration;
+    const client = configuration.newClient({}, { w: 1 });
+
+    client.connect((err, client) => {
+      const db = client.db(configuration.db);
+      const collection = db.collection('arrayfilterstest');
+      const docs = [{ a: [{ x: 1 }, { x: 2 }] }, { a: [{ x: 3 }, { x: 4 }] }];
+      const close = e => client.close(() => done(e));
+      collection.insertMany(docs).then(() =>
+        collection.updateMany(
+          {},
+          { $set: { 'a.$[i].x': 5 } },
+          { arrayFilters: [{ 'i.x': 5 }] },
```

Comment:
Will this array filter match anything? It looks like none of the elements above have `x: 5`.

Reply:
It first set x:5 using $set: { 'a.$[i].x': 5 }

Reply:
Can we add an assertion that that is the case in the callback?